src/kvstore/raftex/RaftexService.cpp

```diff
@@ -94,6 +148,10 @@ void RaftexService::stop() {
         LOG(INFO) << "All partitions have stopped";
     }
     server_->stop();
+
+    // reclaim resource
+    waitUntilStop();
```

Comment:
Does set `status_ ` to `STATUS_STOPPING`  is needed ?

Reply:
yes, I will fix it, thx.

Reply:
yes, I will fix it, thx.